packages/network-controller/tests/provider-api-tests/shared-tests.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable jest/require-top-level-describe, jest/no-export, jest/no-identical-title, jest/no-if */
 
+import { NetworkType } from '@metamask/controller-utils/src';
```

Comment:
Are we exporting this type from `controller-utils`? If so, I think we can drop the `/src`.

Reply:
oh... weird not sure how that happened!